### PR TITLE
Added Tutorial: Install Zammad Community on Linux Based Operating Systems with Docker

### PR DESCRIPTION
**Pull request by JanDragon**

### CODE REVIEW

File: tutorials/install-zammad-community-on-linux-server-with-docker/01.en.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+We need to acces the opt directory with the following command:
+
+`cd /opt`
+
+After that we can download Zammand from the offical GitHub reponsitory.
+
+`git clone https://github.com/zammad/zammad-docker-compose.git`
+
+Then we have to go inside the downloaded folder and set the vm.max_map_count:
```
Comment (n-se) [inline tutorials/install-zammad-community-on-linux-server-with-docker/01.en.md:64]: Could you mention why this is required?

File: tutorials/install-zammad-community-on-linux-server-with-docker/01.en.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+## Step 5 - Start Zammad
+
+Now we just need to start Docker:
+
+`docker-compose up`
+
+
+## Conclusion
+
+Now you can access the web panel by your server IP address or your domain. There you have to create an admin account and then you can start helping people.
```
Comment (n-se) [inline tutorials/install-zammad-community-on-linux-server-with-docker/01.en.md:79]: Does it actually run on port 80? I dont think so.
Also you should mention and / or link to a tutorial on how to setup a reverse proxy with TLS.

File: tutorials/install-zammad-community-on-linux-server-with-docker/01.de.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+`sysctl -w vm.max_map_count=262144`
+
+## Schritt 5 - Start Zammad
+
+Jetzt müssen wir nur noch Docker starten:
+
+`docker-compose up`
+
+
```
Comment (BackInBash) [inline tutorials/install-zammad-community-on-linux-server-with-docker/01.de.md:75]: Wie sieht es mit einem **Schritt 6 - Aktualisierung** aus?

Gerade bei den ganzen Docker apps wird das schnell unter den Tisch gekehrt...

**Comment by MrGeneration** [comment]
The tutorial should -in my opinion- also provide information on required cpu cores. This will help administrators choosing the correct sizing for their installations and not to run into performance issue that easy. ( https://docs.zammad.org/en/latest/prerequisites/hardware.html )

Also, this doc is missing the quite important note/suggestion not to use latest tag, so for completness, you'd proberbly want to mention that ( https://docs.zammad.org/en/latest/install/docker-compose.html ) - when you're at that point, you might also want to point out on how/where to change this version if needed: https://github.com/zammad/zammad-docker-compose/blob/master/.env

File: tutorials/install-zammad-community-on-linux-server-with-docker/01.de.md (hunk 3 above)
Comment (MrGeneration) [reply tutorials/install-zammad-community-on-linux-server-with-docker/01.de.md:75]: Updating documentation currently is a bit off and doesn't contain crucial warnings for about the next 1-2 weeks. However, there are important notes when updating from specific versions.

These remarks can be found here:
https://github.com/zammad/zammad-docker-compose#upgrading

**Comment by n-se** [comment]
Do you still plan to work on this tutorial?

**Comment by rook59** [comment]
Closing due to inactivity. If you plan to update your tutorial feel free to submit a new pull request.

**Pull request closed without merging**